fsw/cfe-core/src/sb/cfe_sb_api.c

```diff
@@ -1320,12 +1320,7 @@ int32  CFE_SB_SendMsgFull(CFE_SB_Msg_t    *MsgPtr,
                 RtgTblPtr->SeqCnt);
     }/* end if */
 
-    /* store the sender information */
-    if(CFE_SB.SenderReporting != 0)
-    {
-       BufDscPtr->Sender.ProcessorId = CFE_PSP_GetProcessorId();
-       strncpy(&BufDscPtr->Sender.AppName[0],CFE_SB_GetAppTskName(TskId,FullName),OS_MAX_API_NAME);
-    }
+    BufDscPtr->SenderId = CFE_SB.AppId;
```

Comment:
Doesn't this mean "SenderId" is always the CFE_SB AppID?

Reply:
> Doesn't this mean "SenderId" is always the CFE_SB AppID?

Yes.